docs/MachineConfigController.md

```diff
@@ -45,8 +45,16 @@ type MachineConfigPoolSpec struct {
     // This includes generating new desiredMachineConfig and update of machines.
     Paused bool `json:"paused"`
 
-    // MaxUnavailable specifies the percentage or constant number of machines that can be updating at any given time.
-    // default is 1.
+    // maxUnavailable defines either an integer number or percentage
+    // of nodes in the corresponding pool that can go Unavailable during
```

Comment:
nit, this is a MachineConfigPoolSpec property, so "in the pool" is probably sufficient without needing "corresponding".